### PR TITLE
added email documentation and assets

### PR DESCRIPTION
**Pull request by CrystalOnScript**
Pushed up email documentation for learn, develop and integrate.

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+</head>
+```
+
+
+
+
+## AMP Specific Tag Replacements 
+
+Since the AMP for Email library is a sub-sect of the AMP HTML library, many of the same rules apply; AMP specific tags replace resource heavy HTML tags and require a defined width and height. This allows the AMP boilerplate to hide content until it has an idea of how it looks on the user's device.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: `sub-sect`?

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+*   read-write
+*   required
+*   valid
+
+
+## Media query features
+
+
+
+*   pointer
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: I don't understand what this means.

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md [hunk 3]
```diff
@@ -183,9 +183,7 @@ In Gmail, AMP email supports attributes and pseudo-classes disallowed in other e
 
 ## Media query features
 
-
-
-*   pointer
+*   [pointer](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer)
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: Still don't understand why this is under Media Query features?

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
+---
+$title: AMP for Email Fundamentals 
+$order: 1
+author: CrystalOnScript
+formats:
+  - email
+---
+
+If you're familiar with AMP, great news! AMP for Emails is just a subsect of the AMP HTML library. If you're unfamiliar with AMP, also great news! This guide will give you everything you need to know to get started writing valid AMP Emails!
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: Is subsect the right word here? The only meaning I could find is [this one](https://www.merriam-webster.com/dictionary/subsect), but this only refers to sects (as in cults). Imo subset would be a better term.

Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: Lol, good catch. I can't believe that slipped past me... twice....

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+Email providers who support AMP Emails have set up security checks to ensure users get a delightful and safe experience. An email build with AMP must meet all requirements:
+
+
+
+*   Start with the `<!doctype html>` doctype. This is also standard for HTML.
+*   Contain a top-level a `<html amp4email>` tag, or an `<html ⚡4email>` tag if your email is extra cool. This identifies the document as an AMP Email so it can be treated as such.
+*   Define both `<head>` and `<body>` tags. This is optional in HTML, but AMP keeps things pristine!
+*   Include a `<meta charset="utf-8>` tag as the first child of the `<head>` tag. This identifies the encoding for the page.
+*   The AMP library is imported through a `<script async src="https://cdn.ampproject.org/v0.js"></script>` tag placed in the `<head>`. Without it, none of the awesome and dynamic functionality gained through AMP will work! As a best practice, this should be included as early as possible in the `<head>`, directly under the `<meta charset="utf-8">` tag.
+*   Initially hide the email content until the AMP library is loaded by placing the AMP for Email boilerplate in the `<head>`.
```
Comment (fstanis) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: Nit: this phrasing makes it sound like the boilerplate is optional. I'd go with:

"The AMP for Email boilerplate inside `<head>` ensures the email content stays hidden until the AMP library is loaded."

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+  }
+</style>
+...
+</head>
+```
+[tip type="important"]
+    AMP enforces a size limit of 50,000 bytes for styling.
+[/tip]    
+
+In Gmail, AMP email supports attributes and pseudo-classes disallowed in other email MIME types.
```
Comment (fstanis) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: I'm not sure if we should include something Gmail-specific here, but I'm fine with keeping it until we find it a better home.

Comment (fstanis) [reply pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md]: Also, I think we should mention that the AMP validator, for most part, won't catch CSS attributes / pseudo-classes that are not allowed. Instead, the email client just strips or ignores these.

**Pull request merged**